packages/icon-button/src/IconButton.tsx

```diff
@@ -205,27 +143,102 @@ const activeStyle: { readonly [K in Variant]: string } = {
       background-color: ${uiColors.gray.dark2};
     }
   `,
-};
+} as const;
 
-const getIconStyle = (size: Size) => css`
+const iconStyle = css`
   position: absolute;
   top: 0;
   bottom: 0;
   left: 0;
   right: 0;
-  margin: auto;
-  height: ${sizeMap[size]}px;
-  width: ${sizeMap[size]}px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 `;
 
+interface IconProps extends React.SVGProps<SVGSVGElement> {
+  glyph: string;
+  size?: Size | number;
+  title?: string | null | boolean;
+}
```

Comment:
Since applications can't yet tree-shake, we're duplicating this interface from the types in the namespaces within the Icon package rather than importing the Icon package.

Reply:
Great comment! May want to inline this comment to preserve it in the code base.

Reply:
Which comment in particular are you talking about? There's no comment where you left this.

Reply:
Oh, what you wrote on GitHub:

> Since applications can't yet tree-shake, we're duplicating this interface from the types in the namespaces within the Icon package rather than importing the Icon package.

Reply:
🤦‍♂ Right, will do hah